tests/python/codegen/test_target_codegen_riscv.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import tvm
+from tvm import te
+from tvm.target.codegen import target_has_features
+
+
+@tvm.testing.requires_llvm_minimum_version(14)
+@tvm.testing.parametrize_targets(
+    "llvm -device=riscv_cpu -mtriple=riscv32-linux-gnu -mcpu=generic-rv32 -mattr=+i,+m",
+    "llvm -device=riscv_cpu -mtriple=riscv32-linux-gnu -mcpu=generic-rv32 -mattr=+i,+m,+v",
+    "llvm -device=riscv_cpu -mtriple=riscv64-linux-gnu -mcpu=generic-rv64 -mattr=+64bit,+a,+c,+d,+f,+m",
+    "llvm -device=riscv_cpu -mtriple=riscv64-linux-gnu -mcpu=generic-rv64 -mattr=+64bit,+a,+c,+d,+f,+m,+v",
+)
+def test_rvv(target):
+    def check_rvv_presence(N):
+        K = te.size_var("K")
+        A = te.placeholder((K, N), dtype="int8", name="A")
+        B = te.placeholder((K, N), dtype="int8", name="B")
+        k = te.reduce_axis((0, K))
+        C = te.compute(
+            (N,),
+            lambda n: te.sum(A[k, n].astype("int32") * B[k, n].astype("int32"), axis=[k]),
+            name="C",
+        )
+        s = te.create_schedule(C.op)
+        s[C].vectorize(s[C].op.axis[0])
```

Comment:
we are moving away from te schedule very soon, so let us instead use tir schedule

Reply:
Reworked, it is even more simple now, I was expecting ```relay``` to go, not realizing the same about ```te```.

Reply:
te compute will stay, but we wanted to remove te.schedule